src/mlpack/core/data/split_data.hpp

```diff
@@ -176,15 +294,24 @@ std::tuple<arma::Mat<T>, arma::Mat<T>, arma::Row<U>, arma::Row<U>>
 Split(const arma::Mat<T>& input,
       const arma::Row<U>& inputLabel,
       const double testRatio,
-      const bool shuffleData = true)
+      const bool shuffleData = true,
+      const bool stratifyData = false)
```

Comment:
We can resolve the issue by adding documentation for  `stratifyData` here.

Reply:
Thanks for pointing that out!. Made the fix.